test/web-platform-tests/to-upstream/dom/nodes/move-me-plz.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
I would put them in https://github.com/w3c/web-platform-tests/tree/master/html/semantics/forms/the-input-element for input-related ones (including anything about checked) and https://github.com/w3c/web-platform-tests/tree/master/html/dom/elements/elements-in-the-dom for anything specific to HTMLElement. Name isn't terribly important.